civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -307,10 +367,19 @@ function civihr_employee_portal_init() {
     drupal_add_css(drupal_get_path('module', 'civihr_employee_portal') . "/lib/tablesaw/tablesaw.css");
 
     _add_script_to_page('tasks.js', 'dashboard');
+    _add_script_to_page('/js/scripts.js');
```

Comment:
Why don't you change the function so that it optionally accepts a list of scripts?

```php
_add_script_to_page('tasks.js', 'dashboard');
_add_script_to_page([
  '/js/filters.js',
  '/js/plupload/plupload.full.min.js',
  '/js/chart.min.js'
]);